pkg/scheduler/apis/config/types.go

```diff
@@ -288,7 +288,7 @@ type Extender struct {
 	// ManagedResources is a list of extended resources that are managed by
 	// this extender.
 	// - A pod will be sent to the extender on the Filter, Prioritize and Bind
-	//   (if the extender is the binder) phases iff the pod requests at least
+	//   (if the extender is the binder) phases if the pod requests at least
```

Comment:
Hi @panli889 
FYI: `iff` means `if and only if`
https://simple.wikipedia.org/wiki/If_and_only_if